enterprise/frontend/src/embedding/auth-common/validate-session-token.ts

```diff
@@ -1,35 +1,46 @@
 import * as MetabaseError from "embedding-sdk-bundle/errors";
 
-export function validateSessionToken(session: any) {
-  if (!session || typeof session !== "object") {
+export function validateSessionToken(token: any) {
```

Comment:
We're literally passing token, the name even says so it's `session token` not `session`

Reply:
Fixed by 2fac8aba48f